src/components/molecules/card/card.scss

```
@@ -0,0 +1,51 @@
+@use "../../variables" as *;
+
+.card {
+  position: relative;
+  border-radius: 0;
+  border: none;
+  background-color: $white_smoke;
+  .card-title {
+    position: absolute;
+    left: 6px;
+    z-index: 1;
+    font-size: $fontsize-l;
+    font-weight: 700;
+    line-height: 29px;
+    color: $orange_bistre;
+
+    &::before {
+      position: absolute;
+      left: -6px;
+      z-index: -1;
+      border-bottom: 150px solid transparent;
+      border-left: 150px solid white;
+      content: "";
+    }
+  }
+
+  .card-image {
+    overflow: hidden;
+    img {
+      width: 100%;
+      transition: all 0.5s ease;
```

今回のように効いてほしいプロパティが限定されている場合は、このように書くと思わぬ副作用が出づらいのでお勧めです。
```suggestion
      transition: transform 0.5s ease;
```